<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFieldEditDialogClass</class>
 <widget class="QDialog" name="QFieldEditDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFieldEditDialog</string>
  </property>
  <widget class="QPushButton" name="cmdOK">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>680</width>
     <height>340</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
